infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutKmPostDao.kt

```diff
@@ -111,12 +112,11 @@ class LayoutKmPostDao(
         val params =
             mapOf(
                 "track_number_ids" to trackNumberIds.map { id -> id.intValue },
-                "design_id" to branch.designId?.intValue,
                 // listOf(null) to indicate an empty list due to SQL syntax limitations; the "is
                 // distinct from true" checks
                 // explicitly for false or null, since "foo in (null)" in SQL is null
                 "km_post_ids_to_publish" to (kmPostIdsToPublish.map { id -> id.intValue }.ifEmpty { listOf(null) }),
-            )
+            ) + target.sqlParameters()
```

Comment:
Toisaalta tykkään tästä `sqlParameters()`-mallista aika paljon (se on simppeli, yhtenäisyyttä tuova rakenne, ja `sqlParameters` sisältää selkeästi nimenomaan `ValidationTarget`:in juttuja,) mutta toisaalta se on kyllä aika erilainen rakenne kuin mitä meillä on ollut muualla (plus jos niissä parametreissa on jotain ongelmia (esim. väärä parametrin nimi,) niin pelkästään tätä Dao-funktiota tutkimalla ei saa selville mikä on pielessä) 🤔 Muilla mielipiteitä asian tiimoilta?

Reply:
Sarjassamme "en keksi parempaakaan", mutta onhan tuo vähän erilainen rakenne verrattuna aiempiin. Saattaa jäädä ehkä enemmän huomaamattomaksi tuon toisen mappi-instanssin luontilistauksen jälkeen, että aa sinne tuleekin 0-N kappaletta vielä lisää asioita, jotka teoriassa voivat kai myös ylikirjoittaa noita tässä yhteydessä listattuja?